web/src/rows.ts

```diff
@@ -126,7 +126,7 @@ export function get_closest_group(element: string): JQuery {
     return $(element).closest("div.recipient_row");
 }
 
-export function get_closest_row(element: string): JQuery {
+export function get_closest_row(element: HTMLElement): JQuery {
```

Comment:
`get_closest_group` seems to have the same bug; do you want to audit this file for such issues and do a prep commit fixing them all?

Reply:
I didn't found any irregularities in the file apart from the one you mentioned.